src/Simulation/Simulators.Tests/TestProjects/Directory.Build.props

```diff
@@ -0,0 +1,9 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This is a great idea! Would it perhaps make sense to apply this Directory.Build.props one more level up still, so that it applies to everything in Simulators.Tests?

Reply:
I considered that, but looking at the rest of the tests they all use Simulators.Tests.props, so already have documentation suppressed. I realize this leaves us open to making this mistake again if a new folder with projects that don't use a common props file is introduced, but hopefully that is easier to spot. Overall, I think accidentally publishing documentation for test structures is the lesser evil compared with accidentally skipping publication of library/runtime documentation, so I like the publish by default model in this repo, and we just need to watch out for places in the tree that are known to have multiple projects that are test-only and follow one of the two patterns (common props file or directory props file).